[stable/airflow] improve git-sync.sh

thesuperzapper opened the pull request:
This fixes a crash as git pull will not always succeed (as in the case of a detached HEAD from forced commit).
Additionally, this will remove any files that have been added to the DAG folder, which is a good security practice for Airflow.

gsemet reviewed (commented):
Ah yes you have to update the chart version

thesuperzapper committed:
[stable/airflow] improve git-sync.sh

This fixes a crash as git pull will not always succeed (as in the case of a detached HEAD from forced commit).
Additionally, this will remove any files that have been added to the DAG folder, which is a good security practice for Airflow.

Signed-off-by: Mathew Wicks <thesuperzapper@users.noreply.github.com>

gsemet reviewed (approved):
/lgtm
